javacutil/src/org/checkerframework/javacutil/InternalUtils.java

```diff
@@ -247,6 +248,19 @@ public static boolean isCaptured(TypeVariable typeVar) {
     }
 
     /**
+     * If typeVar is a captured wildcard, returns that wildcard; otherwise returns null.
+     */
+    public static WildcardType getCapturedWildcard(TypeVariable typeVar) {
+        if (isCaptured(typeVar)) {
+            if (typeVar instanceof AnnotatedType) {
```

Comment:
I think you can simplify this and always call `unannotatedType`, it does no harm.
If you want to special case, the `instanceof` should be a Kind check.